agent/jvm/src/main/java/org/jolokia/jvmagent/JolokiaServerConfig.java

```diff
@@ -451,7 +451,7 @@ private void initAuthenticatorFromAuthMode() {
         } else {
             // Multiple auth strategies were configured, pass auth if any of them
             // succeed.
-            authenticator = new MultiAuthenticator(MultiAuthenticator.Mode.ANY, authenticators);
+            authenticator = new MultiAuthenticator(MultiAuthenticator.Mode.fromString(jolokiaConfig.get(ConfigKey.AUTH_MATCH)), authenticators);
```

Comment:
btw, if the config is null an error is thrown by fromString. I changed this to return ANY for a none-config (as this was the default before).